spec/models/perseverance_scraper_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe PerseveranceScraper, type: :model do
+	let!(:perseverance) { create(:rover, name: "Perseverance", landing_date: Date.new(2021, 2, 18), launch_date: Date.new(2020, 7, 30), status: "active") }
```

Comment:
This looks like it was accidentally indented.

Reply:
The python person in me :) Fixed.